### ISSUE TITLE
[nextgen-editor] twig inside editor

### ISSUE THREAD

**Issue by xf-** [getgrav/grav-premium-issues#124]
Based on https://learn.getgrav.org/17/cookbook/twig-recipes#list-the-last-5-recent-blog-posts

Saving it with complete foreach results in a lot of `<twig>` and to simplify debugging, I put it into a partial.

###  Without wrapper in `.md`
```twig
<twig>{% include 'partials/blog-last-two-items.html.twig' %}</twig>
```
1. Issue outputs <twig> in FE
2. Saving results in removing <twig>
3. Removed twig makes messy output - half escaped

### Wrapped with HTML and unwrapped twig section
```twig
<div class="raw-html-embed">
	<div class="bg-secondary text-white  mt-5 py-5">
		{% include 'partials/blog-last-two-items.html.twig' %}
        </div>
</div>
```
1. The open and close gets wrapped `<twig>{%</twig>`
2. Exception in FE

### Wrap with HTML and twig
```twig
<div class="raw-html-embed">
	<div class="bg-secondary text-white  mt-5 py-5">
		<twig>{% include 'partials/blog-last-two-items.html.twig' %}</twig>
	</div>
</div>
```
1. twig tag is in FE, everything else works. `<twig>` in FE results in layout issues

## Tried and works
```twig
{% include 'partials/blog-last-two-items.html.twig' %}
```
1. Easier to mess up in the editor.
2. Need to put the wrapper inside the partial - less reusable

**The Code from the Code won't work.**

**Comment by w00fz** [comment]
This is fixed and will be available in v1.1.4

**Comment by w00fz** [comment]
v1.1.4 has been released FYI, should be available shortly in GPM
